### PR TITLE
Add Ben Sullivan to participants folder

### PR DESCRIPTION
**Pull request by benterprise**
- Create new yml file with drsloth/Ben Sullivan information.
- Fill in my details

### CODE REVIEW

File: _data/oss_workshop/participants/drsloth.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: Ben Sullivan
+emoji: 'tophat'
```
Comment (dleve123) [inline _data/oss_workshop/participants/drsloth.yml]: This emoji sucks.

Comment (benterprise) [inline _data/oss_workshop/participants/drsloth.yml]: Disagree

**Comment by dleve123** [comment]
Generally, this looks great, but please change the emoticon

**Comment by dleve123** [comment]
Merging :ship:

**Comment by benterprise** [comment]
Sweet

**Pull request merged**